File: chaincode/dto/users.dto.go
package dto

type CreateUserDto struct {
	Id    string `json:"id"`
	IdKey string `json:"idkey"`
	Name  string `json:"name"`
	Email string `json:"email"`
}
type UpdateUserDto struct {
	Id    string `json:"id"`
	IdKey string `json:"idkey"`
	Name  string `json:"name"`
	Email string `json:"email"`
}

File: dapp/src/dto/users.dto.ts
export interface CreateUserDto {
  name: string
  email: string
  idkey: string
}
export interface UpdateUserDto {
  id: number
  name?: string
  email?: string
  idkey?: string
}
export interface CreateCCUserDto {
  id: string
  idkey: string
  name: string
  email: string
}
export interface UpdateCCUserDto {
  id: string
  idkey: string
  name: string
  email: string
}
